go.mod

```diff
@@ -185,5 +185,6 @@ require (
 replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.8.0-sdk-v0.46.7
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.15.0-tm-v0.34.23.0.20230308212354-07aba19d0bda
+	// TODO remove this replace once we merge dependent PR and cut a celestia-core release
+	github.com/tendermint/tendermint => ../celestia-core
```

Comment:
can we point to a commit here so we can see failing tests here and in our own local repos?